cypress/integration/chi-js/modal.spec.js

```diff
@@ -0,0 +1,182 @@
+describe('chi-modal', function() {
+  before(() => {
+    cy.visit('http://localhost:8000/tests/js/modal.html');
+  });
+
+  describe('check base modal', () => {
+    it(`modal should be opened when we click launch demo modal button`, () => {
+      cy.get('button#modal-trigger-1').click();
+      cy.get('#modal-1')
+        .should('have.class', '-active')
+        .find('.-close')
+        .click()
+        .wait(550);
+    });
+
+    it(`modal should be closed when we click close button`, () => {
+      cy.get('button#modal-trigger-1').click();
+      cy.get('#modal-1')
+        .find('.-close')
+        .click()
+        .wait(550)
+        .should('be.not.visible');
+    });
```

Comment:
```suggestion
        .should('have.class', '-active');
    });

    it(`modal should be closed when we click close button`, () => {
      cy.get('#modal-1')
        .find('.-close')
        .click()
        .wait(550)
        .should('be.not.visible');
    });
```

Reply:
No need for closing the modal and reopening it in the next test case. Let's leave it open and close it in the next one only. It will run faster this way.

Reply:
Please do the same for the next test cases as well

Reply:
Fixed in the latest commit